### PR TITLE
Iht 4321 Registration Content Changes

### PR DESCRIPTION
**Pull request by tyagikumarvineet**

### CODE REVIEW

File: app/iht/views/registration/kickout/iht_kickout_template.scala.html [hunk 1]
```diff
@@ -26,7 +26,7 @@
 hasTimeOut=hasTimeOut) {
 
     <div class="panel-indent">
-        <p id="kickout-message"><strong>@summaryMessage</strong></p>
+        <p id="kickout-message">@summaryMessage</p>
     </div>
```
Comment (dmbeer) [inline app/iht/views/registration/kickout/iht_kickout_template.scala.html]: remove blank lines

File: app/iht/views/registration/kickout/kickout_template.scala.html [hunk 2]
```diff
@@ -25,19 +25,17 @@
...
 
 <div class="panel-indent">
-    <p id="kickout-message"><strong>@summary</strong></p>
+    <p id="kickout-message">@summary</p>
 </div>
 
     <h2 class="heading-medium">@Messages("iht.nextSteps")</h2>
-    @contentLines.map { line => <p>@line</p> }
+    @contentLines.map { line => <p>@line@if(line == contentLines.last){@Messages("iht.comma")<a id="return-button" href="@returnToUrl">
+                                        @Messages("iht.registration.kickout.returnToTheLastPageVisited")</a>}</p> }
```
Comment (dmbeer) [inline app/iht/views/registration/kickout/kickout_template.scala.html]: blank lines

File: app/iht/views/registration/kickout/iht_kickout_template.scala.html (hunk 1 above)
Comment (tyagikumarvineet) [inline app/iht/views/registration/kickout/iht_kickout_template.scala.html]: Done

File: app/iht/views/registration/kickout/kickout_template.scala.html (hunk 2 above)
Comment (tyagikumarvineet) [inline app/iht/views/registration/kickout/kickout_template.scala.html]: Done

**Pull request merged**